lib/fennec/turn.ex

```diff
@@ -3,11 +3,13 @@ defmodule Fennec.TURN do
   This module defines a struct used as TURN protocol state.
   """
 
-  defstruct allocation: nil, permissions: [], channels: []
+  defstruct allocation: nil, permissions: [], channels: [], nonce: "", realm: ""
```

Comment:
I'd make defaults for `:nonce` and `:realm` `nil`, this way they won't leak out of server empty if we forget to assign them (as additional safety net).